libs/form-configuration/core/test/loader/fetch-form-configurations.test.ts

```diff
@@ -17,20 +17,21 @@
 
 import { fetchFormConfigurations } from "../../src/loader";
 import { useFormConfigurationStore } from "../../src/store";
-import { mockFormConfigurations, mockValidatorConverters } from "../testutil/form-configuration-generating-util";
+import { mockFormConfigurations, mockFormYupConfigurations, mockValidatorConverters } from "../testutil/form-configuration-generating-util";
 import { setupFormConfigurationServer } from "../testutil/setup-form-configuration-server";
 
 const server = setupFormConfigurationServer();
 
 beforeAll(() => {
+  useFormConfigurationStore.getState().set([mockFormYupConfigurations[0]]);
```

Comment:
please add another test and set this in "given" part, instead of reusing this test